code/datums/api.dm

```diff
@@ -880,15 +880,7 @@ proc/api_update_command_database()
 		reportannounce = 1
 
 	//Send the message to the communications consoles
-	for (var/obj/machinery/computer/communications/C in SSmachinery.processing_machines)
-		if(! (C.stat & (BROKEN|NOPOWER) ) )
-			var/obj/item/weapon/paper/P = new /obj/item/weapon/paper( C.loc )
-			P.name = "[command_name()] Update"
-			P.info = reportbody
-			P.update_space(P.info)
-			P.update_icon()
-			C.messagetitle.Add("[command_name()] Update")
-			C.messagetext.Add(P.info)
+	post_comm_message("[command_name()] Update", reportbody)
```

Comment:
I just noticed three things here:
On line 876:
Can you replace`
reporttitle = "NanoTrasen Update"
`with`
reporttitle = "[command_name()] Update"
`
Use reporttitle in post_comm_message() for the title

And move post_comm_message before
`if(reportannounce == 1)`
because the messagebody gets modified in 
`if (reporttype == "ccia")`